packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/NewUpdateManifest.kt

```diff
@@ -58,6 +66,7 @@ class NewUpdateManifest private constructor(
           mLaunchAsset.getNullable("fileExtension")
         ).apply {
           url = Uri.parse(mLaunchAsset.getString("url"))
+          headers = assetHeaders[mLaunchAsset.getString("key")]
```

Comment:
looks like this field is used for response headers in iOS, so we'll need to create a new field for these called requestHeaders for use in both Android and iOS.